packages/office-ui-fabric-react/src/components/Image/Image.Props.ts

```diff
@@ -27,6 +27,12 @@ export interface IImageProps extends React.HTMLProps<HTMLImageElement> {
    * If true, the image frame will expand to fill its parent container.
    */
   maximizeFrame?: boolean;
+
+  /**
+   * Optional callback method for when the image is loaded or is in error.
+   * The 'success' parameter indicates whether the loading was successful or in error.
+   */
+  onImageLoad?: (success: boolean) => void;
```

Comment:
I did initially, but was unsure how it would react to a broken link or other loading error. I figured that as long as `Image` had that logic anyway, why not expose and re-use it?